src/tools/rbd_mirror/Throttler.cc

```diff
@@ -0,0 +1,204 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "Throttler.h"
```

Comment:
Ah, good point. I forgot about this. I will change it (return back) to ImageSyncThrottler.